### PR TITLE
Task two completed

### PR DESCRIPTION
**Pull request by rbln1**
Второе задание, включая бонусную часть с цветами расходов/доходов, выполнено.

### CODE REVIEW

File: app/src/main/java/me/rubl/loftmoney/screens/adapters/BudgetPagerAdapter.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            case 1:
+                return BudgetFragment.newInstance(ItemModel.BudgetType.INCOME);
+
+            default: return null;
+        }
+    }
+
+    @Override
+    public int getCount() {
+        return 2;
```
Comment (askden8) [inline app/src/main/java/me/rubl/loftmoney/screens/adapters/BudgetPagerAdapter.java]: don't use "magic numbers"

File: app/src/main/java/me/rubl/loftmoney/screens/adapters/ItemsAdapter.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        void bind(ItemModel itemModel) {
+            mNameTV.setText(itemModel.getName());
+            mValueTV.setText(itemModel.getValue().concat(" ₽"));
+
+            // TODO: get value text color from resources
+
+            switch (itemModel.getType()) {
+
+                case EXPENSE:
+                    mValueTV.setTextColor(Color.parseColor("#4a90e2"));
```
Comment (askden8) [inline app/src/main/java/me/rubl/loftmoney/screens/adapters/ItemsAdapter.java]: use colors from colors.xml

File: app/src/main/java/me/rubl/loftmoney/screens/fragments/BudgetFragment.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+    private RecyclerView mRecyclerMain;
+    private FloatingActionButton mFabMain;
+
+    private List mData;
+
+    public static BudgetFragment newInstance(ItemModel.BudgetType budgetType) {
+        BudgetFragment instance = new BudgetFragment();
+        Bundle args = new Bundle();
+        args.putSerializable("budget_type", budgetType);
```
Comment (askden8) [inline app/src/main/java/me/rubl/loftmoney/screens/fragments/BudgetFragment.java]: use constants

File: app/src/main/java/me/rubl/loftmoney/screens/fragments/BudgetFragment.java [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+import me.rubl.loftmoney.screens.adapters.ItemsAdapter;
+import me.rubl.loftmoney.screens.model.ItemModel;
+
+public class BudgetFragment extends Fragment {
+
+    private ItemsAdapter mItemsAdapter = new ItemsAdapter();
+    public static final int ADD_ITEM_REQUEST = 100;
+
+    //TODO
+    public ItemModel.BudgetType BUDGET_TYPE = ItemModel.BudgetType.INCOME;
```
Comment (askden8) [inline app/src/main/java/me/rubl/loftmoney/screens/fragments/BudgetFragment.java]: why with big letters?

File: app/src/main/res/layout/activity_add_item.xml [hunk 5]
```diff
@@ -1,28 +1,48 @@
...
-        android:layout_height="wrap_content"
-        android:layout_marginBottom="8dp"
-        android:inputType="numberDecimal"
-        android:hint="@string/add_item_price_hint" />
+        <EditText
+            android:id="@+id/et_add_item_name"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:minHeight="48dp"
+            android:textSize="24sp"
```
Comment (askden8) [inline app/src/main/res/layout/activity_add_item.xml]: pls use styles

File: app/src/main/res/layout/activity_add_item.xml [hunk 6]
```diff
@@ -1,28 +1,48 @@
...
     <Button
-        android:layout_width="200dp"
+        android:id="@+id/btn_add_item"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_gravity="center_horizontal"
+        android:layout_gravity="bottom|center_horizontal"
+        style="?android:attr/borderlessButtonStyle"
+        android:drawableStart="@drawable/ic_done_black_24dp"
+        android:drawableTint="#7ed321"
```
Comment (askden8) [inline app/src/main/res/layout/activity_add_item.xml]: use colors from colors xml

File: app/src/main/java/me/rubl/loftmoney/screens/adapters/BudgetPagerAdapter.java (hunk 1 above)
Comment (rbln1) [reply app/src/main/java/me/rubl/loftmoney/screens/adapters/BudgetPagerAdapter.java]: Corrections made.
`private static final int NUM_PAGES = 2;
@Override
public int getCount() {
return NUM_PAGES;
}`

File: app/src/main/java/me/rubl/loftmoney/screens/fragments/BudgetFragment.java (hunk 4 above)
Comment (rbln1) [reply app/src/main/java/me/rubl/loftmoney/screens/fragments/BudgetFragment.java]: Corrections made.
`private static final int ADD_ITEM_REQUEST = 100;
    private ItemModel.BudgetType mBudgetType;`

File: app/src/main/java/me/rubl/loftmoney/screens/fragments/BudgetFragment.java (hunk 3 above)
Comment (rbln1) [reply app/src/main/java/me/rubl/loftmoney/screens/fragments/BudgetFragment.java]: Corrections made.
`public static final String KEY_BUDGET_TYPE = "budget_type";`

File: app/src/main/res/layout/activity_add_item.xml (hunk 6 above)
Comment (rbln1) [reply app/src/main/res/layout/activity_add_item.xml]: Corrections made.
`android:drawableTint="@color/accept_color"`

File: app/src/main/res/layout/activity_add_item.xml (hunk 5 above)
Comment (rbln1) [reply app/src/main/res/layout/activity_add_item.xml]: Corrections made.
`android:minHeight="@dimen/et_add_item_min_height" android:textSize="@dimen/et_add_item_text_size"`

File: app/src/main/java/me/rubl/loftmoney/screens/adapters/ItemsAdapter.java (hunk 2 above)
Comment (rbln1) [reply app/src/main/java/me/rubl/loftmoney/screens/adapters/ItemsAdapter.java]: Corrections made.
Solved by transferring the context to the bind method of holder.
`void bind(ItemModel itemModel, Context context) {...}`

**Pull request merged**